content/enterprise_influxdb/v1.9/administration/config-data-nodes.md

```diff
@@ -228,6 +228,20 @@ InfluxData recommends values ranging from `0ms` to `100ms` for non-SSD disks.
 
 Environment variable: `INFLUXDB_DATA_WAL_FSYNC_DELAY`
 
+#### `ingress-metric-by-measurement-enabled = false`
+
+When `true`, collect statistics of points written per-measurement.
+These can be accessed via the `/debug/vars` endpoint and in the `_internal` database if that is configured.
+
+Environment variable: `INFLUXDB_DATA_INGRESS_METRIC_BY_MEASUREMENT_ENABLED`
+
+#### `ingress-metric-by-login-enabled = false`
+
+When `true`, collect statistics of points written per-login.
+These can be accessed via the `/debug/vars` endpoint and in the `_internal` database if that is configured.
+
+Environment variable: `INFLUXDB_DATA_INGRESS_METRIC_BY_LOGIN_ENABLED`
```

Comment:
Are these environment variables correct, @lesam ?

Reply:
Yes they are. We don't do the env variables individually, so as long as they follow the right pattern they will override the right piece of config.